libbeat/docs/shared-feature-flags.asciidoc

```diff
@@ -1,16 +1,24 @@
 [[configuration-feature-flags]]
-== Configure Feature Flags
+== Configure feature flags
 
 ++++
-<titleabbrev>Feature Flags</titleabbrev>
+<titleabbrev>Feature flags</titleabbrev>
 ++++
 
-Example configuration with the FQDN feature flag enabled:
+The Feature Flags section of the +{beatname_lc}.yml+ config file contains
+settings in {beatname_uc} that are disabled by default. These may include 
+experimental features, changes to behaviors within {beatname_uc} or its 
+components, or settings that could cause a breaking change. For example a
+setting that changes information included in events might be inconsistent with
+the naming pattern expected in your configured {beatname_uc} output.
```

Comment:
```suggestion
The Feature Flags section of the +{beatname_lc}.yml+ config file contains
settings in {beatname_uc} that are disabled by default. These may include 
experimental features, changes to behaviors within {beatname_uc}, or 
settings that could cause a breaking change. For example a
setting that changes information included in events might be inconsistent with
the naming pattern expected in your configured {beatname_uc} output.
```

Agent has components that it supervises, e.g. Beats.  Beats, however, don't have components in the same sense.

Reply:
Oh, of course! Thanks for catching that.